cmd/new/new.go

```diff
@@ -56,10 +56,14 @@ func CreateCommand() *cobra.Command {
 		return utils.Find(formats, toComplete), cobra.ShellCompDirectiveNoSpace | cobra.ShellCompDirectiveNoFileComp
 	})
 
-	home, _ := homedir.Dir()
 	tmp.Flags().StringVar(&localTemplateCache, "templatepath", "", "location of installed templates")
 	viper.BindPFlag("templatepath", tmp.Flags().Lookup("templatepath")) //nolint:errcheck
-	viper.SetDefault("templatepath", filepath.Join(home, ".pdk", "pct"))
+	execDir, err := osext.ExecutableFolder()
+	if err != nil {
+		log.Fatal().Msgf("Could not obtain bin working dir path: %v", err)
+		os.Exit(1)
```

Comment:
return error.New(custom msg) and it'll handle thee exit code and return

Reply:
In the end, given our conversation, I simplified the change:
- The `new.go` package is simply to build the command - it now just configures the default and optionally user provided value(s) for `templatepath`
- If one or both of those directories happen to be empty / contain invalid content templates, they'll now get [a warning](https://github.com/puppetlabs/pdkgo/pull/41/files#diff-f91432ef2a97bebe3295799af49783df3328da73b7f55d6aec8c636aedab5e29R94-R96)